olp-cpp-sdk-core/tests/geo/tiling/TileKeyTest.cpp

```diff
@@ -311,82 +311,72 @@ TEST(TileKeyTest, parentChildRelationIsTransitive) {
   }
 }
 
-// tests for GetNearestAvailableTileKeyLevel(...)
-TEST(TileKeyTest,
-     getNearestAvailableTileKeyLevel_emptyLevelSet_returnedOptionalIsEmpty) {
-  TileKeyLevels levels;
+TEST(TileKeyTest, GetNearestAvailableTileKeyLevel) {
+  {
+    TileKeyLevels levels;
```

Comment:
```suggestion
    SCOPED_TRACE("Empty level set - return empty");
    TileKeyLevels levels;
```

For all others also. Add SCOPED_TRACE so that we know what is tested and what the outcome is.
Optionally a comment.